<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="5" tests="14" time="147.086">
 <testsuite name="careers jnj automation sctipts" timestamp="2018-06-10T22:59:31" hostname="localhost" time="147.086" errors="0" tests="14" skipped="3" disabled="0" failures="5">
  <testcase classname="careers jnj automation sctipts" name="index page UI elements checks" time="11.165" />
  <testcase classname="careers jnj automation sctipts" name="should verify position search" time="12.298" />
  <testcase classname="careers jnj automation sctipts" name="should verify location search" time="0">
   <skipped message="Temporarily disabled with xit" />
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should verify position details page" time="18.188" />
  <testcase classname="careers jnj automation sctipts" name="position details page UI elements checks" time="0">
   <skipped message="Temporarily disabled with xit" />
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should verify get_alert_field positive validation" time="7.722" />
  <testcase classname="careers jnj automation sctipts" name="should verify position Details Page Apply button" time="0">
   <skipped message="Temporarily disabled with xit" />
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should verify login Page " time="18.342" />
  <testcase classname="careers jnj automation sctipts" name="should login and check uploadResume page" time="16.273" />
  <testcase classname="careers jnj automation sctipts" name="should check the Select From Prev Resume" time="5.76">
   <failure type="exception" message="Failed: waits for selectFromPrevResumeButton
Wait timed out after 5740ms"><![CDATA[TimeoutError: waits for selectFromPrevResumeButton
Wait timed out after 5740ms
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
    at ManagedPromise.invokeCallback_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: waits for selectFromPrevResumeButton
    at scheduleWait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at uploadResumePage._callee$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\uploadResumePage.js:35:40)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\uploadResumePage.js:3:191)
From: Task: Run it("should check the Select From Prev Resume") in control flow
    at UserContext.<anonymous> (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:405:5)
    at addSpecsToSuite (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:22:1)
    at Module._compile (module.js:652:30)
    at loader (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:565:32)]]>
   </failure>
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should check the cover letter page" time="6.138">
   <failure type="exception" message="Failed: waits for lblattachLetter
Wait timed out after 5077ms"><![CDATA[TimeoutError: waits for lblattachLetter
Wait timed out after 5077ms
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
    at ManagedPromise.invokeCallback_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: waits for lblattachLetter
    at scheduleWait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at attachCoverLetterPage._callee$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\attachCoverLetterPage.js:28:44)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\attachCoverLetterPage.js:3:191)
From: Task: Run it("should check the cover letter page") in control flow
    at UserContext.<anonymous> (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:436:5)
    at addSpecsToSuite (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:22:1)
    at Module._compile (module.js:652:30)
    at loader (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:565:32)]]>
   </failure>
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should check the personal information page" time="1.306">
   <failure type="exception" message="Failed: No element found using locator: By(xpath, //div[@class=&apos;mat-form-field-infix&apos;]/input[@placeholder=&apos;First Name&apos;])"><![CDATA[NoSuchElementError: No element found using locator: By(xpath, //div[@class='mat-form-field-infix']/input[@placeholder='First Name'])
    at elementArrayFinder.getWebElements.then (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as clear] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as clear] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at personalIndoPage._callee$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:27:39)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:3:191)
    at C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:3:437
    at new Promise (<anonymous>)Error
    at ElementArrayFinder.applyAction_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as sendKeys] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as sendKeys] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at personalIndoPage._callee$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:27:47)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:3:191)
    at C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:3:437
    at new Promise (<anonymous>)
From: Task: Run it("should check the personal information page") in control flow
    at UserContext.<anonymous> (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:456:5)
    at addSpecsToSuite (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:22:1)
    at Module._compile (module.js:652:30)
    at loader (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:565:32)]]>
   </failure>
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should check the personal information page" time="5.89">
   <failure type="exception" message="Failed: waits for personalInfoPage
Wait timed out after 5013ms"><![CDATA[TimeoutError: waits for personalInfoPage
Wait timed out after 5013ms
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
    at ManagedPromise.invokeCallback_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: waits for personalInfoPage
    at scheduleWait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at personalAddressPage._callee$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalAddressPage.js:40:44)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalAddressPage.js:3:191)
From: Task: Run it("should check the personal information page") in control flow
    at UserContext.<anonymous> (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:480:5)
    at addSpecsToSuite (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:22:1)
    at Module._compile (module.js:652:30)
    at loader (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:565:32)]]>
   </failure>
  </testcase>
  <testcase classname="careers jnj automation sctipts" name="should check the Email and Phone" time="27.255">
   <failure type="exception" message="Failed: waits for personalAddressPage
Wait timed out after 5092ms"><![CDATA[TimeoutError: waits for personalAddressPage
Wait timed out after 5092ms
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
    at ManagedPromise.invokeCallback_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: waits for personalAddressPage
    at scheduleWait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at personalIndoPage._callee2$ (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:52:40)
    at tryCatch (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-polyfill\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (C:\JibeAutomation_own\jibe0106\ForAutomation\objects\PageObjects\personalInfoPage.js:3:191)
From: Task: Run it("should check the Email and Phone") in control flow
    at UserContext.<anonymous> (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:524:5)
    at addSpecsToSuite (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\vahan.bakaryan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\JibeAutomation_own\jibe0106\ForAutomation\tests\suites\applyJobTests.js:22:1)
    at Module._compile (module.js:652:30)
    at loader (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (C:\JibeAutomation_own\jibe0106\ForAutomation\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (module.js:565:32)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>